docs/en/wallet-instance-attestation.rst

```diff
@@ -2,56 +2,676 @@
+|                                   | instance attestation issuer. (see |
+|                                   | example below)                    |
++-----------------------------------+-----------------------------------+
+| sub                               | The public url of the issuer      |
+|                                   | concatenated with the thumbprint  |
+|                                   | of the JWK of the wallet instance |
+|                                   | for which the attestation is      |
+|                                   | being issued                      |
++-----------------------------------+-----------------------------------+
+| iat                               | Timestamp of issue of the         |
+|                                   | attestation                       |
++-----------------------------------+-----------------------------------+
+| exp                               | Attestation expiration timestamp. |
+|                                   | A good practice to avoid security |
+|                                   | problems is to have a limited     |
+|                                   | duration of the attestation.      |
++-----------------------------------+-----------------------------------+
+| type                              | String:                           |
+|                                   | "WalletInstanceAttestation"       |
++-----------------------------------+-----------------------------------+
+| policy_uri                        | Url to the privacy policy         |
+|                                   | of the wallet                     |
++-----------------------------------+-----------------------------------+
+| tos_uri                           | Url to the terms                  |
+|                                   | of use of the wallet provider     |
++-----------------------------------+-----------------------------------+
+| logo_uri                          | Logo url of the wallet provider   |
++-----------------------------------+-----------------------------------+
+| asc                               | Attested security context:        |
+|                                   | Represents a level of "trust" of  |
+|                                   | the service containing a Level Of |
+|                                   | Agreement defined in the metadata |
+|                                   | of the wallet provider            |
++-----------------------------------+-----------------------------------+
+| cnf                               | This parameter contains the JSON  |
+|                                   | object called jwk containing the  |
+|                                   | public key of the Wallet          |
+|                                   | Instance, necessary for the       |
+|                                   | holder binding                    |
++-----------------------------------+-----------------------------------+
```

Comment:
I think that in the attestation we should put also wallet capabilities as defined here
https://openid.bitbucket.io/connect/openid-4-verifiable-presentations-1_0.html#section-8.1-3